SETUP_DOCKER.md

```diff
@@ -1,36 +1,41 @@
-# Getting started using Docker
+## Setting up a complete development environment from scratch
 
-To set up a local development environment in Docker, you have to build a new Docker image for
-tol-lab-share. start a stack of services that include a Redpanda schema registry and Rabbitmq. 
-You can do all together by running the command:
-
-```shell
-docker-compose up -d
+1. Start dependent services: Rabbitmq and Redpanda
+```bash
+  cd docker
+  docker-compose up -d
+  cd ..
```

Comment:
Up to you, but I put an `up.sh` script and a `down.sh` script in the `dependencies` folder for Crawler and Lighthouse which does all the folder changing for you.  Better yet, it works when called from any folder because it uses the script file location as the one to run the `docker-compose` command from.

```
#!/bin/bash

cd "$(dirname "$0")"
docker-compose up -d
```